runtime/tests/interpreter/integers_test.go

```diff
@@ -882,3 +883,53 @@ func TestInterpretIntegerMinMax(t *testing.T) {
 		})
 	}
 }
+
+func TestStringIntegerConversion(t *testing.T) {
+
+	for _, typ := range append(sema.AllSignedIntegerTypes, sema.AllUnsignedIntegerTypes...) {
+		numericType := typ.(*sema.NumericType)
+		low := numericType.MinInt()
+		if low == nil {
+			low = big.NewInt(0)
+		}
+		high := numericType.MaxInt()
+		if high == nil {
+			high = big.NewInt(math.MaxInt64)
+		}
+
+		typeName := typ.String()
+
+		t.Run(typeName, func(t *testing.T) {
+
+			code := fmt.Sprintf(`
+				fun testFromString(_ input: String): Int? {
+					return %s.fromString(input).map(Int)
```

Comment:
Maybe remove the `.map(Int)` function, to reduce the number of things we test in a single test-case. Or make it (calling `map` on the returned value) a separate test.
```suggestion
					return %s.fromString(input)
```

Reply:
hmm, my reason for converting everything to an Int was to reduce the amount of manual conversions for expected values. since Int has an unbounded domain, this approach only needs to specify low and high values.